src/main/java/carRacing/model/WinnersName.java

```diff
@@ -27,7 +27,7 @@ private int findRecords(List<RacingCar> racingCars){
 
     private String findWinnersName(int highScore, RacingCar car){
         if (highScore == car.position()) {
```

Comment:
```suggestion
        if (car.isSamePosition(highScore)) {
```
값을 꺼내서 비교하지 말고
car 에게 직접 메시지를 전달하면 좋겠습니다